main.go

```diff
@@ -32,11 +32,38 @@ const (
 	// DefaultServiceDescription is the default service description.
 	DefaultServiceDescription = "HashiCorp Vault Service Broker"
 
-	// DefaultPlanName is the name of our plan, only one supported
+	// DefaultPlanName is the name of our plan, only one is currently supported
 	DefaultPlanName = "shared"
 
 	// DefaultPlanDescription is the default description.
 	DefaultPlanDescription = "Secure access to Vault's storage and transit backends"
+
+	//DefaultDisplayName is the default name of the service to be displayed in graphical clients.
+	DefaultDisplayName = "Vault for PCF"
```

Comment:
I switched over to using a different method for reading env variables and setting defaults, because that makes it easier to read the config and also I can iterate it to add support for Credhub. My apologies for the bother! 

Would you mind switching over to making these fields [like this](https://github.com/hashicorp/vault-service-broker/blob/master/main.go#L153)?

Reply:
I see the new one! Thank you!